<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>niftk::IGIVideoOverlayWidget</class>
 <widget class="QWidget" name="niftk::IGIVideoOverlayWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>991</width>
    <height>494</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QCheckBox" name="m_LeftImageCheckBox">
       <property name="layoutDirection">
        <enum>Qt::RightToLeft</enum>
       </property>
       <property name="text">
        <string>left</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QmitkDataStorageComboBoxWithSelectNone" name="m_LeftImageCombo"/>
     </item>
     <item>
      <widget class="QCheckBox" name="m_RightImageCheckBox">
       <property name="layoutDirection">
        <enum>Qt::RightToLeft</enum>
       </property>
       <property name="text">
        <string>right</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QmitkDataStorageComboBoxWithSelectNone" name="m_RightImageCombo"/>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="m_TrackingLabel">
       <property name="text">
        <string>tracking</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QmitkDataStorageComboBoxWithSelectNone" name="m_TrackingCombo"/>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="m_OpacityLabel">
       <property name="text">
        <string>opacity</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="m_OpacitySlider">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>200</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="m_TrackedViewCheckBox">
       <property name="text">
        <string>tracked</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="m_3DViewCheckBox">
       <property name="text">
        <string>3D</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QSplitter" name="m_Splitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QWidget" name="m_OverlayAndTrackedViewer">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="niftk::SingleVideoWidget" name="m_LeftOverlayViewer" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
       <item>
        <widget class="niftk::SingleVideoWidget" name="m_RightOverlayViewer" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
       <item>
        <widget class="niftk::SingleVideoWidget" name="m_TrackedViewer" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QmitkRenderWindow" name="m_3DViewer" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QmitkRenderWindow</class>
   <extends>QWidget</extends>
   <header>QmitkRenderWindow.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QmitkDataStorageComboBoxWithSelectNone</class>
   <extends>QComboBox</extends>
   <header location="global">QmitkDataStorageComboBoxWithSelectNone.h</header>
  </customwidget>
  <customwidget>
   <class>niftk::SingleVideoWidget</class>
   <extends>QWidget</extends>
   <header>niftkSingleVideoWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
